.github/workflows/release-latest.yml

```diff
@@ -28,6 +28,22 @@ jobs:
       - run: |
           git tag v${{ fromJson(steps.pkg.outputs.json).version }}
           git push origin v${{ fromJson(steps.pkg.outputs.json).version }}
+      - name: Set up Docker Buildx
```

Comment:
This is not part of npm publish, so it should be in a separate job. Also, it should be moved to a separate action so that it can be used in the 2.x release as well.

Reply:
Tbh this is my workaround for not knowing how to have jobs dependent on one another :cry:. I'll have another look.

Reply:
You can use `needs: ['publish']` similar to the `docs` job.